src/input.cpp

```diff
@@ -527,8 +533,8 @@ static void input_mapping_execute_matching_or_generic(bool allow_commands) {
             continue;
         }
 
-        if (m.seq.length() == 0) {
-            generic = &m;
+        if (m.seq == L"\xE000") {
+          generic = &m;
```

Comment:
FYI, I'm a style nazi :smile: Please run `make style` against your changes (you can do it before or after you run `git commit`) to ensure the code follows the project style.